test/scripts/e2e-cp.sh

```diff
@@ -200,5 +202,10 @@ go test -test.timeout 180m -v . \
         -kubeconfig ${KUBECONFIG} \
         ${NUM_NODES:+"--num-nodes=${NUM_NODES}"} \
         ${E2E_REPORT_DIR:+"--report-dir=${E2E_REPORT_DIR}"} \
-        ${E2E_REPORT_PREFIX:+"--report-prefix=${E2E_REPORT_PREFIX}"}
+        ${E2E_REPORT_PREFIX:+"--report-prefix=${E2E_REPORT_PREFIX}"} || {
+            sudo df -h
```

Comment:
does this need to be temporary? seems like low overhead and never know when it could be useful down the road. either way, if someone found it and hated it they can track down the reasoning with `git blame`.

no strong opinion here though. I can add a TODO if you want

Reply:
ack I am ok with us leaving it here if that was a goal; I do think it helps for sure to catch stuff and its harmless. ok resolving this comment; you have convinced me :)